src/dials/util/image_viewer/slip_viewer/frame.py

```diff
@@ -203,16 +203,16 @@ def handle_position_event(self, event):
             )
 
             posn_str = "Picture:  fast={:.3f} / slow={:.3f} pixels.".format(
-                fast_picture,
-                slow_picture,
+                fast_picture + 0.5,
```

Comment:
Kind of a lazy choice to add the offset here- rather than fix the function that produces the coordinates. I briefly looked into changing some of the position functions, namely via the `view_offset` parameters, in pyslip.py, but this wasn't as straightforward as I had hoped and I reverted to the lazy code change.

Reply:
Lazy correction to add 0.5 offset in pix location lines 206-207 and 215. This restores enforces the convection that the origin be located at the corners (0,0).

Reply:
Updated to only adjust coords[0] and [1], and to remove the extra half pixel add a bit later.  Comment moved and rewritten to reflect the above converations